### ISSUE TITLE
WISH:  color names and values should go in `system/catalog` or in `system/view`

### ISSUE THREAD

**Issue by luce80** [red/REP#131]
I wish that color names and their values would be located somewhere else then in `system/words` so I can simply find and/or list them or one of them.

**Comment by hiiamboris** [comment]
Good idea.
Meanwhile you can use @9214's opus magnum:
`exclude sort extract load help-string tuple! 2 [transparent glass]`

**Comment by luce80** [comment]
Yes, I found that. But I was searcing for a script that was able to show the color names because I didn't found them when I searched in `system` and never know the `help tuple!` trick. Next time will I remember the `help` trick, the @9214 trick or the other script name ?
